grunt/index.d.ts

```diff
@@ -796,8 +796,8 @@ declare namespace grunt {
              *
              * @note taskFunction.apply(scope: grunt.task.IMultiTask<any>, args: any[])
              */
-            registerMultiTask(taskName: string, taskFunction: Function): void
-            registerMultiTask(taskName: string, taskDescription: string, taskFunction: Function): void
+            registerMultiTask(taskName: string, taskFunction: (this: IMultiTask<any>, ...args: any[]) => void): void;
+            registerMultiTask(taskName: string, taskDescription: string, taskFunction: (this: IMultiTask<any>, ...args: any[]) => void): void;
```

Comment:
@jeffmay that would work. Though, my concern is just that it will breaks peoples code. Since not providing a type (there is also noway TS can infer the type) the default type will be `{}`. 

What we want though is default generic types, though I'm not sure when that will happen:
https://github.com/Microsoft/TypeScript/issues/2175